Source/JavaScriptCore/runtime/StringReplaceCache.h

```diff
@@ -30,14 +30,11 @@
 
 namespace JSC {
 
-DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(StringReplaceCache);
-
 class JSImmutableButterfly;
 class RegExp;
 
 class StringReplaceCache {
-    WTF_MAKE_FAST_ALLOCATED;
-    WTF_MAKE_NONCOPYABLE(StringReplaceCache);
+
```

Comment:
This seems unnecessary.

Reply:
Re-added. Was able to resolve with `DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER_AND_EXPORT`.